desktop/src/main/java/bisq/desktop/main/portfolio/closedtrades/ClosedTradesView.java

```diff
@@ -218,19 +272,23 @@ protected void activate() {
         exportButton.setOnAction(event -> {
             final ObservableList<TableColumn<ClosedTradableListItem, ?>> tableColumns = tableView.getColumns();
             CSVEntryConverter<ClosedTradableListItem> headerConverter = transactionsListItem -> {
-                String[] columns = new String[7];
+                String[] columns = new String[11];
                 for (int i = 0; i < columns.length; i++)
                     columns[i] = tableColumns.get(i).getText();
 
                 return columns;
             };
             CSVEntryConverter<ClosedTradableListItem> contentConverter = item -> {
-                String[] columns = new String[7];
+                String[] columns = new String[11];
                 columns[0] = model.getTradeId(item);
                 columns[1] = model.getDate(item);
                 columns[2] = model.getAmount(item);
                 columns[3] = model.getPrice(item);
                 columns[4] = model.getVolume(item);
+				columns[5] = model.getTxFee(item);
+				columns[6] = model.getMakerFee(item);
+				columns[7] = model.getBuyerSecurityDeposit(item);
+				columns[8] = model.getSellerSecurityDeposit(item);
                 columns[5] = model.getDirectionLabel(item);
```

Comment:
there is something wrong with the indexing here. you added 5,6,7,8, yet, the last lines reassign 5 and 6. Ultimately resulting in a
```
java.lang.NullPointerException
	at com.googlecode.jcsv.writer.internal.CSVColumnJoinerImpl.joinColumns(CSVColumnJoinerImpl.java:30)
	at com.googlecode.jcsv.writer.internal.CSVWriterImpl.write(CSVWriterImpl.java:38)
	at com.googlecode.jcsv.writer.internal.CSVWriterImpl.writeAll(CSVWriterImpl.java:29)
	at bisq.desktop.util.GUIUtil.exportCSV(GUIUtil.java:266)
```
because 9 and 10 stay unassigned (null).

Reply:
Ah. Apologies. This is fixed.